scss/abstracts/_variables.scss

```diff
@@ -51,3 +52,34 @@
   /* Simple Card */
   --simple-card-constrained-width: 12.2rem;
 }
+
+// Sass Variables
+$color-primary: #6070ff;
+
+$breakpoints: (
+  // 15.625em = 250px
+  'screen-xxsm': 15.625em,
+  // 20em = 320px
+  'screen-xsm': 20em,
+  // 31.25em = 500px
+  'screen-sm': 31.25em,
+  // 50em = 800px
+  'screen-md': 50em,
+  // 62.5em = 1000px
+  'screen-lg': 62.5em,
+  // 68.75em = 1100px
+  'screen-xlg': 68.75em,
+  // 75em = 1200px
+  'screen-xxlg': 75em,
+  // 81.25em = 1300px
+  'screen-hg': 81.25em,
+  // 93.75em = 1500px
+  'screen-xhg': 93.75em,
+  // 100em = 1600px
+  'screen-xxhg': 100em
+);
```

Comment:
- [ ] **[REQUIRED CHANGES]**

- Please, kindly remove unused code, it makes your codebase not clean.